src/platform/user/profile/vet360/tests/containers/AddressValidationModal.unit.spec.jsx

```diff
@@ -397,14 +399,14 @@ describe('<AddressValidationModal/>', () => {
         .find('label')
         .at(1)
         .text(),
-    ).to.equal('12345 1st Ave bldg 2 apt 23 Tampa, FL 12346');
+    ).to.equal('12345 1st Ave, bldg 2, apt 23Tampa, FL 12346');
```

Comment:
Did we mean to not have a space between `23` and `Tampa`?

Reply:
Tampa is rendered on a new line 😄

Reply:
No big deal but I'm curious if there's a way to check for `\n` or something?